generate_workspace/src/main/java/com/google/devtools/build/workspace/GenerateWorkspaceOptions.java

```diff
@@ -53,6 +53,12 @@
   public List<String> mavenProjects = new ArrayList<>();
 
   @Parameter(
+          names = {"--repository", "-r"},
+          description = "Repository to query for artifacts."
```

Comment:
Nit. Can you replace `repository` with `repositories` or `repository_list`?
And can you replace the description so that it is
```
List of remote maven repositories to query for artifacts
```